ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetObjectVersions.java

```diff
@@ -0,0 +1,48 @@
+package com.spectralogic.ds3cli.command;
+
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3cli.ArgumentFactory;
+import com.spectralogic.ds3cli.Arguments;
+import com.spectralogic.ds3cli.View;
+import com.spectralogic.ds3cli.ViewType;
+import com.spectralogic.ds3cli.models.GetObjectVersionsResult;
+import com.spectralogic.ds3cli.util.pagination.GetObjectVersionsLoaderFactory;
+import com.spectralogic.ds3cli.views.cli.GetObjectVersionsView;
+import com.spectralogic.ds3cli.views.json.DataView;
+import com.spectralogic.ds3client.models.S3Object;
+import com.spectralogic.ds3client.utils.collections.LazyIterable;
+import org.apache.commons.cli.Option;
+
+public class GetObjectVersions extends CliCommand<GetObjectVersionsResult> {
+
+    private static final ImmutableList<Option> requiredArgs = ImmutableList.of(ArgumentFactory.BUCKET, ArgumentFactory.OBJECT_NAME);
+
+    private String bucketName;
+    private String objectName;
```

Comment:
missing `final`?

Reply:
This cannot be final because it's not assigned until after the constructor has been called.  You'll find that this is consistent with the other Commands.